File: C#/백준/Bronze/15025. Judging Moose/Judging Moose.cs
using System;

namespace q{
    class w{
        static void Main(){
            string[] aa = Console.ReadLine().Split();
            int a = int.Parse(aa[0]);
            int b = int.Parse(aa[1]);
            if(a==b){
                if(a==0){
                    Console.WriteLine("Not a moose");
                }
                else{
                    Console.WriteLine($"Even {a*2}");
                }
            }
            else{
                int c = a>b?a:b;
                Console.WriteLine($"Odd {c*2}");
            }
        }
    }
}

File: Java/백준/Bronze/15025. Judging Moose/Judging Moose.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a =in.nextInt();
        int b = in.nextInt();
        if(a==b){
            if(a==0){
                System.out.println("Not a moose");
            }
            else{
                System.out.printf("Even %d",a*2);
            }
        }
        else{
            int c = a>b?a:b;
            System.out.printf("Odd %d",c*2);
        }
    }
}